go/src/koding/klientctl/metrics/metrics.go

```diff
@@ -0,0 +1,76 @@
+package metrics
+
+import (
+	"context"
+	"path/filepath"
+
+	kitesconfig "koding/kites/config"
+
+	statsd "github.com/DataDog/datadog-go/statsd"
+)
+
+// Metrics wraps metric
+type Metrics struct {
+	bolt    *BoltQueue
+	Datadog *statsd.Client
+}
+
+// New creates new metrics collector.
+func New() (*Metrics, error) {
+	boltPath := filepath.Join(kitesconfig.KodingHome(), "metrics.bolt")
```

Comment:
since we have only one datadog service all the metrics will eventually go there, for resolving the issue i was planning to send the endpoint in the metrics tags, ie:

```config_show_timing:1.639387|ms|#commandName:config_show,rootCommandName:config,subCommandName:show,debug:false,endpoints_ipCheck_public:https://p.koding.com/-/ipcheck,endpoints_ip_public:https://p.koding.com/-/ip,endpoints_kdLatest_public:https://koding-kd.s3.amazonaws.com/production/latest-version.txt,endpoints_klientLatest_public:https://koding-klient.s3.amazonaws.com/production/latest-version.txt,endpoints_klient_private:http://127.0.0.1:56789/kite,endpoints_koding_private:http://127.0.0.1,endpoints_koding_public:https://koding.com,endpoints_tunnel_public:http://t.koding.com/kite,teamName:team,success:true```

so we can do the filtering on datadog UI